core/data/pack/README.MD

```diff
@@ -1,63 +1,42 @@
-# Proto-Pack format Version 1.1
+# Proto-Pack format Version 2.0
```

Comment:
After months of attempting to find a compromise between designs, I believed we had settled on a format we were all happy with. Why are you proposing changing this now?

**There needs to be a really compelling argument to change this format again.**

Reply:
Well.  I am mostly just trying to simplify it.
It takes me a while to parse the 1.1 format due to all the various sub blocks and +/-1 offsets.
It does not really affect the spirit of the chunks, and it would be minimal changes to the encoder/decoder.
The other things I propose are file format breaking anyway, so might as well try to tweak it so I am happy with it as well.

Reply:
It'll take you less time to think about the existing 1.1 format than it would to restart this discussion again, to justify why you're trying to push thread-concepts into a file-protocol, reimplement the encoders and decoders, [redesign file level versioning](https://github.com/google/gapid/blob/master/core/data/pack/pack.go), rework [Dan's optimizations](https://github.com/google/gapid/pull/1133), rewrite [the tests](https://github.com/google/gapid/blob/master/core/data/pack/pack_test.go), and reimplement the `gapit unpack` command.

> The other things I propose are file format breaking anyway, so might as well try to tweak it so I am happy with it as well.

Not a very compelling argument so far, I'm afraid.

Reply:
Well, I am not really trying to push thread-concepts in.  I happy to drop that part and disallow/reserve positive parent values.  I added that though in maybe cause I realized it fits in nicely and saves space.

Reply:
We're weeks away from trying to get a V1 release out. There are plenty of more important things for us to be looking at right now than nitting over something that is working fine for us.

Reply:
I do not think the rest of the code would change as significantly.  It is mostly just reordering fields removing the +1/-1 special cases.

Yeah, but after the V1 release, there is going to be argument for preserving backwards compatibility,  so this is the only to make tweaks.